bin/millau/node/src/chain_spec.rs

```diff
@@ -164,7 +164,7 @@ fn testnet_genesis(
 ) -> GenesisConfig {
 	GenesisConfig {
 		frame_system: SystemConfig {
-			code: WASM_BINARY.to_vec(),
+			code: WASM_BINARY.expect("Millau development WASM not available").to_vec(),
```

Comment:
Tbh I'm not sure how this can happen. I tried `SKIP_WASM_BUILD=1 cargo run -p rialto-bridge-node -- --dev` but nothing terrible happened 🤷‍♂️